packages/pwa-kit-create-app/assets/bootstrap/js/config/default.js.hbs

```diff
@@ -10,17 +10,32 @@ module.exports = {
     app: {
         // Customize how your 'site' and 'locale' are displayed in the url.
         url: {
+            {{#if answers.project.demo.enableDemoSettings}}
             // Determine where the siteRef is located. Valid values include 'path|query_param|none'. Defaults to: 'none'
-            // site: 'none',
+            site: 'path',
             // Determine where the localeRef is located. Valid values include 'path|query_param|none'. Defaults to: 'none'
-            locale: 'none'
+            locale: 'path',
             // This boolean value dictates whether or not default site or locale values are shown in the url. Defaults to: false
-            // showDefaults: true
+            showDefaults: true,
+            {{else}}
+            // Determine where the siteRef is located. Valid values include 'path|query_param|none'. Defaults to: 'none'
+            site: 'none',
+            // Determine where the localeRef is located. Valid values include 'path|query_param|none'. Defaults to: 'none'
+            locale: 'none',
+            // This boolean value dictates whether or not default site or locale values are shown in the url. Defaults to: false
+            showDefaults: false,
+            {{/if}}
+            // This boolean value dictates whether the plus sign (+) is interpreted as space for query param string. Defaults to: false
+            interpretPlusSignAsSpace: false
```

Comment:
Cool, thanks. It looks like last time we missed adding `interpretPlusSignAsSpace` to this file.